platform/ios/MGLMapboxEvents.m

```diff
@@ -16,6 +16,7 @@
 static const NSUInteger version = 1;
 static NSString *const MGLMapboxEventsUserAgent = @"MapboxEventsiOS/1.0";
 static NSString *MGLMapboxEventsAPIBase = @"https://api.tiles.mapbox.com";
+static BOOL usingTestServer = NO;
```

Comment:
Actually, since it isn't constant, let's make it a property or ivar on MGLMapboxEvents: usesTestServer.